usaspending_api/awards/v2/filters/award_filters.py

```diff
@@ -0,0 +1,42 @@
+"""
+For Award V2 Contract/Assistance Download, the filters are quite trivial and completely unrelated to
+other types of filters so they get their own file.
+"""
+import logging
+
+from usaspending_api.accounts.v2.filters.account_download import generate_treasury_account_query
+from usaspending_api.awards.models import TransactionNormalized
+from usaspending_api.awards.models_matviews import SubawardView
+from usaspending_api.common.exceptions import InvalidParameterException
+
+
+logger = logging.getLogger(__name__)
+
+
+def _get_award_id(filters):
```

Comment:
Could use TinyShield here, save a few lines of code and leverage existing request parsing

Reply:
There is a pre-existing model for award_id
from usaspending_api.common.validator.award import get_internal_or_generated_award_id_model

models = get_internal_or_generated_award_id_model()